### PR TITLE
[DOCU-2827] 3.1 fast-follow: Amazon Linux 2022 support

### PR DESCRIPTION
**Pull request by cloudjumpercat**
Signed-off-by: Diana <75819066+cloudjumpercat@users.noreply.github.com>

### Summary

Added Amazon Linux 2022 to the changelog and table of supported OS.
### Reason

DOCU-2827
### Testing

- View the changelog
- View /gateway/latest/os-support/ to see if Amazon Linux 2022 is listed there

### CODE REVIEW

File: app/_data/tables/os_support.yml [hunk 1]
```diff
@@ -190,4 +190,16 @@
...
   deprecation_date: null
-  eol_link: null
\ No newline at end of file
+  eol_link: null
+- name: Amazon Linux 2022
+  enterprise: true
+  enterprise_versions:
+    first_version: 3.1.x
+    last_version: null
+  oss: false
```
Comment (lena-larionova) [inline app/_data/tables/os_support.yml:199]: I might've been wrong about this now being supported for OSS. There is a kong/kong PR for adding it: https://github.com/Kong/kong/pull/9712 

Let's set it as supported and verify with Colin Hutchinson.

```suggestion
  oss: true
```

File: app/_data/tables/os_support.yml [hunk 2]
```diff
@@ -190,4 +190,16 @@
...
+  oss_versions:
+    first_version: null
```
Comment (lena-larionova) [inline app/_data/tables/os_support.yml:201]:
```suggestion
    first_version: 3.1.x
```

**Review by lena-larionova — COMMENTED** [review]
I recommend reaching out to Colin Hutchinson to double-check availability in OSS for 3.1. I _think_ I was wrong in originally thinking it wouldn't be supported for OSS.

**Review by acgoldsmith — APPROVED** [review]
LGTM

**Pull request merged**